scripts/getFactoryCollection.cdc

```diff
@@ -0,0 +1,5 @@
+import CollectionFactory from "../contracts/CollectionFactory.cdc"
+
+pub fun main(user: String, collectionIDs: {String : [UInt64]}, shard: String) : CollectionFactory.CollectionReport? {
```

Comment:
I could argue that this should only return items. As collections and extraIds are not something you need at this point. Since you already have the collectionIds already. Thoughts?

Reply:
can't agree more. 
I basically return an empty array on collection and empty dictionary for extra IDs